src/main/java/duke/storage/LocalStorage.java

```diff
@@ -0,0 +1,92 @@
+package duke.storage;
+
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
```

Comment:
It is good to see the importing of packages are separated instead of using a single '*'.

Reply:
Thanks!